majasdarbs/src/main/java/io/codelex/loops/practice/RollTwoDice.java

```diff
@@ -0,0 +1,35 @@
+package io.codelex.loops.practice;
+
+import java.util.Scanner;
+
+public class RollTwoDice {
+    public static void main(String[] args) {
+
+        int values = 0;
+        int sum;
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter desired sum: ");
+        sum = scanner.nextInt();
+
+
+        do {
+
+            if (sum > 1 && sum <= 12) {
+
+                int min = 1;
+                int max = 6;
+
+                int getRandomValue = (int) (Math.random() * (max - min)) + min;
```

Comment:
taspats kas iepriekš un tā kā sēit ir divas vienādas rindas, tad labi apsvēŗt izmantot palīgmetodi, piemēŗam, `rollDice()` daudz lasāmāk arī būtu